release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -568,6 +568,80 @@ For more information, see xref:../nodes/scheduling/nodes-descheduler.adoc#nodes-
+
+==== Enable alert routing for user workload monitoring
+A cluster administrator can now enable alert routing for user workload monitoring so that developers and other users can configure custom alerts and alert routing for their user-defined projects.
+
+==== Enable a dedicated Alertmanager instance for user-defined alerts
+You can optionally enable a separate instance of Alertmanager dedicated to sending alerts only for user-defined projects.
+This feature can help reduce the load on the default platform Alertmanager instance and can better separate user-defined alerts from default platform alerts.
+
+==== Use additional authentication settings for remote write configuration
+You can now use the following authentication methods to access a remote write endpoint: AWS Signature Version 4, authorization, and OAuth 2.0. 
+Before this release, you could only use TLS client and basic authentication.
+
+
+==== Create, browse, and manage PromQL queries more easily in the web console
+The Query Browser on the *Observe* -> *Metrics* page of the {product-title} web console adds various enhancements to improve your ability to create, browse, and manage PromQL queries.  
+
+==== Double the scrape interval for ServiceMonitors in single node deployments
+You can now double the scrape interval for all Cluster Monitoring Operator (CMO) controlled ServiceMonitors on single-node {product-title} deployments.
+
+==== Create alerting rules based on platform monitoring metrics (Technology Preview)
+This release introduces a Technology Preview feature in which administrators can create alerting rules based on existing platform monitoring metrics.
+This feature helps administrators to more quickly and easily create new alerting rules specific to their environments.
+
+==== Add cluster ID labels to remote write storage
+You can now add cluster ID labels to metrics being sent to remote write storage.
+You can then query these labels to identify the source cluster for a metric and distinguish that metric data from similar metric data sent by other clusters.
+
+==== Query metrics using the federation endpoint for user workload monitoring
+You can now use the Prometheus `/federate` endpoint to scrape user-defined metrics from a network location outside the cluster.
+Before this release, you could only access the federation endpoint to scrape metrics in default platform monitoring.
+
+==== Enable body size limit for metrics scraping for default platform monitoring
+You can now set the `enforceBodySizeLimit` config map option for default platform monitoring to enable a body size limit on metrics scraping.
+When enabled, this setting can trigger the new `PrometheusScrapeBodySizeLimitHit` alert. The setting also limits the impact that a malicious target can have on both the Prometheus component and on the cluster as a whole.
```

Comment:
The alert `PrometheusScrapeBodySizeLimitHit` is triggered when at least one of Prometheus' scrape targets replies with a response body larger than the configured `enforcedBodySizeLimit`.

the alert only triggered only when the setting for `enforcedBodySizeLimit` is too small

Reply:
Change to:

`This setting triggers the new `PrometheusScrapeBodySizeLimitHit` alert when at least one Prometheus scrape target replies with a response body larger than the configured `enforcedBodySizeLimit`.
`